src/cluster.ts

```diff
@@ -80,6 +90,103 @@ export type GetClusterMetadataCallback = (
   apiResponse?: IOperation | null
 ) => void;
 
+export type BackupTimestamp = google.protobuf.ITimestamp | PreciseDate | Date;
+export interface ModifiableBackupFields {
+  /**
+   * The ITimestamp (Date or PreciseDate will be converted) representing
+   * when the backup will automatically be deleted. This must be at a
+   * minimum 6 hours from the time of the backup request and a maximum of 30
+   * days.
+   */
+  expireTime?: BackupTimestamp;
```

Comment:
`expireTime` could be required here since it is required for creation of a backup, and is the only field that can be updated on an existing backup.